en_US.json

```diff
@@ -2335,6 +2335,7 @@
 	"testsuites.content.discard.running.test": "{cntTestSuites} test suites and {cntTestCases} test cases are running. The test will be discarded.",
 	"testsuites.content.property.associated.info": "The property associated with this test suite is not currently deployed. Please deploy the property or select another property.",
 	"testsuites.content.success.save": "Test case has been successfully updated.",
+	"testsuites.content.successfully.created": "Test suite has been successfully created.",
```

Comment:
I believe it would be better if we begin the sentence with "The".
For instance,   "The test suite has been successfully created."
"The test suite has been successful updated."

"

Reply:
Thanks. I changed some text.